kratos/includes/condition.h

```diff
@@ -63,7 +63,9 @@ class Condition : public GeometricalObject, public Flags
     ///@{
 
     /// Pointer definition of Condition
-    KRATOS_CLASS_POINTER_DEFINITION(Condition);
+    typedef Kratos::intrusive_ptr<Condition> Pointer;
+    typedef Kratos::intrusive_weak_ptr<Condition> WeakPointer;
+    typedef Kratos::unique_ptr<Condition> UniquePointer;
```

Comment:
same

Reply:
the only reason is that i added it manually before adding the macro.

i ll change this tomorrow